static/src/javascripts/projects/common/modules/video/youtube-tracking.js

```diff
@@ -0,0 +1,81 @@
+define([
+    'common/utils/mediator',
+    'lodash/collections/forEach',
+    'common/utils/config',
+    'common/modules/video/ga-helper'
+], function (
+    mediator,
+    forEach,
+    config,
+    gaHelper
+) {
+
+    function eventAction() {
+        return 'video content';
+    }
+
+    function buildEventId(event, videoId) {
+        return event + ':' + videoId;
+    }
+
+    function initYoutubeEvents(videoId) {
+
+        var ga = window.ga,
+            gaTracker = config.googleAnalytics.trackers.editorial;
+
+        var events = {
+            metricMap: {
+                'play': 'metric1',
+                'skip': 'metric2',
+                '25': 'metric3',
+                '50': 'metric4',
+                '75': 'metric5',
+                'end': 'metric6'
+            },
+            baseEventObject: {
+                eventCategory: 'Media',
+                eventAction: 'video content',
+                eventLabel: window.location.pathname,
```

Comment:
we'd want this to be the atom id, to identify each individual atom rather than just the page I think